packages/core/src/runtime/resolveStyleRules.ts

```diff
@@ -88,8 +88,16 @@ export function resolveStyleRules(
 
     const value = styles[property as keyof GriffelStyle];
 
-    // eslint-disable-next-line eqeqeq
-    if (value == null) {
+    if (typeof value === 'undefined') {
+      continue;
+    }
+
+    if (value === null) {
```

Comment:
As discussed offline, let's try to use a special symbol so we don't start introducing different behaviour for undefined vs null

Reply:
@ling1726 after discussion with @george-cz, I went with a a string casted to a CSS value.